kong/plugins/response-transformer/body_transformer.lua

```diff
@@ -56,7 +78,7 @@ local function append_value(current_value, value)
   local current_value_type = type(current_value)
 
   if current_value_type  == "string" then
```

Comment:
style nitpick: remove space before =
```suggestion
  if current_value_type == "string" then
```

Reply:
done, thanks.